packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1189,13 +1190,23 @@ class WidgetsApp extends StatefulWidget {
 
   /// If true, forces the widget inspector to be visible.
   ///
+  /// Overrides the `debugShowWidgetInspector` value set in [WidgetsApp].
+  ///
   /// Used by the `debugShowWidgetInspector` debugging extension.
   ///
-  /// The inspector allows you to select a location on your device or emulator
+  /// The inspector allows the selection of a location on your device or emulator
   /// and view what widgets and render objects associated with it. An outline of
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
-  static bool debugShowWidgetInspectorOverride = false;
+  static bool get debugShowWidgetInspectorOverride {
+    return _debugShowWidgetInspectorOverrideNotifier.value;
+  }
+  static set debugShowWidgetInspectorOverride(bool value) {
+    _debugShowWidgetInspectorOverrideNotifier.value = value;
+  }
+
+  static final ValueNotifier<bool> _debugShowWidgetInspectorOverrideNotifier =
+      ValueNotifier<bool>(false);
```

Comment:
formatting nit: since flutter doesn't have a fixed line limit, we prefer to keep this all on the same line.